R/print.effectsize_table.R

```diff
@@ -2,31 +2,42 @@
 print.effectsize_table <- function(x, digits = 2, ...) {
   x_orig <- x
 
+  footer <- caption <- NULL
+
+  ## Footer
+  ## MSB: Move to own printing function?
```

Comment:
You can tag such comments with `## TODO` or `## FIXME`, and after installing the [todor-package](https://github.com/dokato/todor), you get an _Find package TODOs_ addin (which makes it easy to find all such todo's in a package/project).

Reply:
Oh nice!